src/components/Cards/Stories/BaseCardEqualizerStory.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import {
+  CardBody,
+  CardContainer,
+  CardHeading,
+  CardDropdownButton,
+  CardTitle,
+  CardFooter,
+  CardLink
+} from '../index';
+import {
+  EmptyState,
+  EmptyStateTitle,
+  EmptyStateIcon,
+  EmptyStateInfo
+} from '../../../index';
+import { Icon } from '../../Icon';
+import { MenuItem } from '../../MenuItem';
+import { Row, Col } from '../../Grid';
+import { inlineTemplate } from '../../../../storybook/decorators/storyTemplates';
+import { DOCUMENTATION_URL } from '../../../../storybook/constants';
+
+class SomeExampleName extends React.Component {
+  getNodes(equalizerComponent, equalizerElement) {
+    console.log('NODES:', [this.node1, this.node2, this.node3]);
+    return [this.node1, this.node2, this.node3];
+  }
+
+  render() {
+    const equalized = {
+      display: 'flex',
+      justifyContent: 'space-between',
+      flexDirection: 'column'
+    };
+    const equalizedBody = {
+      flex: 1
+    };
+    return (
+      <Row
```

Comment:
i need some designer help here @jeff-phillips-18 @jgiardino ... should `CardContainer` be wrapped within `Row`? I thought it was typically on the outside...
```
      <Row
        className={'row-cards-pf'}
        style={{ overflow: 'auto', paddingBottom: 100, height: '100%' }}
      >
        <CardContainer nodes={getNodes}>
          <Col xs={6} sm={4} md={4}>
```

Reply:
I did this because i wanted to follow your Equalizer [example](https://github.com/priley86/miq_v2v_ui_plugin/blob/master/app/javascript/migration/scenes/Overview/index.js#L81) exactly as you wrote it so i won't miss anything, i ran into a lot of problems with this package.

Reply:
Yup .. my example was vaporware...It would be good to formalize it now though ;)